lib/corpus.py

```diff
@@ -301,6 +303,172 @@ def _dump_internal(self, out: TextIO, indentation: str = ""):
             out.write("%s</subcorpus>\n" % (indentation,))
 
 
+class CorpusV2(Corpus):
```

Comment:
Does it make sense to derive from `Corpus`?

Reply:
If your comment below is valid, then I shall not inherit from `Corpus`.